test/e2e/apps/deployment.go

```diff
@@ -1122,3 +1122,8 @@ func waitForReplicaSetTargetSpecReplicas(c clientset.Interface, replicaSet *apps
 	}
 	return err
 }
+
+// CheckDeploymentRevisionAndImage checks if the input deployment's and its new replica set's revision and image are as expected.
+func CheckDeploymentRevisionAndImage(c clientset.Interface, ns, deploymentName, revision, image string) error {
```

Comment:
Need to make the function name lower as local function like `checkDeploymentRevisionAndImage` .

Reply:
done

Reply:
@oomichi 
But the test failed again. I don't understand why the test failed, please help me.
Thanks!